content/pt-br/docs/reference/kubectl/cheatsheet.md

```diff
@@ -301,13 +334,46 @@ kubectl run -i --tty busybox --image=busybox -- sh  # Executar pod como shell in
 kubectl run nginx --image=nginx --restart=Never -n 
 mynamespace                                         # Execute o pod nginx em um namespace específico
 kubectl run nginx --image=nginx --restart=Never     # Execute o pod nginx e salve suas especificações em um arquivo chamado pod.yaml
+
```

Comment:
In "## Exclusão de Recursos" add "# Delete a pod with no grace period" like the original text

Here, check the last three commands and the following. It is different from the original text.